src/churn/churn.ts

```diff
@@ -301,6 +301,19 @@ export var filterCandidatesFromSdp = (sdp:string) : string => {
       //     JSON.stringify({
       //       'key': '0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0'
       //     }));
+
+      /*this.pipe_.setTransformer('byteSequenceShaper', undefined,
```

Comment:
Why don't you follow the same convention as AES, i.e. C++-style comments, with our regular formatting.